src/app/hooks/useScreensize.ts

```diff
@@ -0,0 +1,11 @@
+import useMediaQuery from '@mui/material/useMediaQuery'
+import { useTheme } from '@mui/material/styles'
+import { Theme } from '@mui/material/styles/createTheme'
+
+export const useScreenSizeOnTheme = (theme: Theme) => ({
```

Comment:
Do we need this additional hook? If this was created just to handle `ResultListFrame`. I would just replace `isMobile` in styled component with ` [theme.breakpoints.down('sm')]` syntax (no need for calling hook).

Reply:
OK, done as suggested.